packages/client/hmi-client/src/components/models/tera-strata-model-diagram.vue

```diff
@@ -0,0 +1,384 @@
+<template>
+	<main>
+		<TeraResizablePanel v-if="!nodePreview">
+			<div ref="splitterContainer" class="splitter-container">
+				<Splitter :gutterSize="5" :layout="layout">
+					<SplitterPanel
+						class="tera-split-panel"
+						:size="equationPanelSize"
+						:minSize="equationPanelMinSize"
+						:maxSize="equationPanelMaxSize"
+					>
+						<section class="graph-element">
+							<Toolbar>
+								<template #start>
+									<Button
+										@click="resetZoom"
+										label="Reset zoom"
+										class="p-button-sm p-button-outlined toolbar-button"
+									/>
+								</template>
+							</Toolbar>
+							<section class="legend">
+								<ul>
+									<li v-for="(type, i) in stateTypes" :key="i">
+										<div class="legend-state" :style="{ backgroundColor: strataTypeColors[i] }" />
+										{{ type }}
+									</li>
+								</ul>
+								<ul>
+									<li v-for="(type, i) in transitionTypes" :key="i">
+										<div
+											class="legend-transition"
+											:style="{ backgroundColor: strataTypeColors[stateTypes?.length ?? 0 + i] }"
+										/>
+										{{ type }}
+									</li>
+								</ul>
+							</section>
+							<div v-if="model" ref="graphElement" class="graph-element" />
+						</section>
+					</SplitterPanel>
+				</Splitter>
+			</div>
+		</TeraResizablePanel>
+		<div v-else-if="model" ref="graphElement" class="graph-element preview" />
+	</main>
+</template>
+
+<script setup lang="ts">
+import { IGraph } from '@graph-scaffolder/index';
+import { watch, ref, computed, onMounted, onUnmounted, onUpdated } from 'vue';
+import { runDagreLayout } from '@/services/graph';
+import {
+	PetrinetRenderer,
+	NodeData,
+	EdgeData
+} from '@/model-representation/petrinet/petrinet-renderer';
+import { petriToLatex } from '@/petrinet/petrinet-service';
+import {
+	convertAMRToACSet,
+	convertToIGraph
+} from '@/model-representation/petrinet/petrinet-service';
+import Button from 'primevue/button';
+import Splitter from 'primevue/splitter';
+import SplitterPanel from 'primevue/splitterpanel';
+import Toolbar from 'primevue/toolbar';
+import { Model } from '@/types/Types';
+import { strataTypeColors } from '@/utils/color-schemes';
+import TeraResizablePanel from '../widgets/tera-resizable-panel.vue';
+
+// Get rid of these emits
+const emit = defineEmits([
+	'update-tab-name',
+	'close-preview',
+	'asset-loaded',
+	'close-current-tab',
+	'update-model-content'
+]);
+
+const props = defineProps<{
+	model: Model | null;
+	isEditable: boolean;
+	nodePreview?: boolean;
+}>();
+
+const menu = ref();
+
+const newModelName = ref('New Model');
+
+const equationLatex = ref<string>('');
+const equationLatexOriginal = ref<string>('');
```

Comment:
Do we still care about showing LaTex and math equation in this context? Can we just remove them and simplify?

Reply:
I have already removed them for the next pr, if you don't mind waiting until then.